app/frontend/src/components/FileInput.vue

```diff
@@ -408,3 +422,14 @@ export default {
   }
 };
 </script>
+
+<style scoped>
+@media screen and (min-width: 1264px) {
+  .docgen-row .v-card {
+    height: 100%;
+  }
+  .docgen-row {
+    margin-bottom: 30px;
```

Comment:
This should be 1rem instead of 30px - always try to prefer using relative units instead such as em and rem instead of hard values like pixels.